projects/demo/tsconfig.app.json

```diff
@@ -3,6 +3,7 @@
     "files": ["src/main.browser.ts", "src/polyfills.ts"],
     "include": ["**/*.d.ts"],
     "angularCompilerOptions": {
-        "enableIvy": true
+        "enableIvy": true,
+        "strictMetadataEmit": false
```

Comment:
strictMetadataEmit is a flag for libraries. It's not meant to be used for applications.

https://angular.io/guide/angular-compiler-options#strictmetadataemit